<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOutputSet</class>
 <widget class="QDialog" name="DialogOutputSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="3">
    <widget class="QLabel" name="label_station_1_delay_value">
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_output_delay">
     <property name="text">
      <string>吹气持续时间：</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="label_output_delay_value">
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QSpinBox" name="spinBox_station_2_delay">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="label_station_2_delay_value">
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_station_1_output_cam">
     <property name="text">
      <string>工位1输出使用相机：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="comboBox_station_1_output_line"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_port">
     <property name="text">
      <string>port：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QSpinBox" name="spinBox_port">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_ip"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_station_2_delay">
     <property name="text">
      <string>工位2延迟：</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QSpinBox" name="spinBox_output_delay">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="comboBox_station_1_output_cam"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_station_1_delay">
     <property name="text">
      <string>工位1延迟：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="spinBox_station_1_delay">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_ip">
     <property name="text">
      <string>ip地址：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QCheckBox" name="checkBox_station_1_enable">
     <property name="text">
      <string>启用</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_ip_2">
     <property name="text">
      <string>工位2 ip地址：</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="pushButton_station_2_test">
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_station_1_output_line">
     <property name="text">
      <string>工位1输出使用线：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_ip_3">
     <property name="text">
      <string>工位2 port：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_ip_2"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QSpinBox" name="spinBox_port_2">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QPushButton" name="pushButton_station_1_test">
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
